ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/legaladvisor/CCDOrderDirection.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.cmc.ccd.domain.legaladvisor;
+
+import lombok.Builder;
+import lombok.Value;
+
+import java.time.LocalDate;
+
+@Value
+@Builder
+public class CCDOrderDirection {
+    private CCDOrderDirectionType extraOrderDirection;
+    private String otherDirection;
```

Comment:
As there is only one String member,  `other` in this name looks bit misleading here

Reply:
yep :( see https://tools.hmcts.net/jira/browse/ROC-5586